assets/component-cart-notification.css

```diff
@@ -7,17 +7,29 @@
 }
 
 .cart-notification {
+  --shadow-horizontal-offset: var(--popup-drawer-shadow-horizontal-offset);
+  --shadow-vertical-offset: var(--popup-drawer-shadow-vertical-offset);
+  --shadow-blur-radius: var(--popup-drawer-shadow-blur-radius);
+  --shadow-opacity: var(--popup-drawer-shadow-opacity);
+  border-bottom-right-radius: var(--popup-drawer-corner-radius);
+  border-bottom-left-radius: var(--popup-drawer-corner-radius);
   background-color: rgb(var(--color-background));
-  border-color: rgba(var(--color-foreground), 0.2);
+  border-color: rgba(var(--color-foreground), var(--popup-drawer-border-opacity));
   border-style: solid;
-  border-width: 0 0 0.1rem;
+  border-width: 0 0 var(--popup-drawer-border-width);
```

Comment:
on mobile you only get the bottom border but for things like the menu drawer you do have all sides except the left. Doesn't it seem a bit inconsistent ? [screenshot](https://screenshot.click/21-11-w57sk-svogn.png).
I feel like maybe you could have both sides and the bottom border on mobile for the cart notification 🤔

Reply:
I was just following what border was already there for mobile and they are different on mobile and desktop. @melissaperreault thoughts?

Reply:
Ok, I can see where we lack of guidance here. Lets wait until we explore these components deeper 🙏 
cc. @danielvan